packages/react/vite.config.ts

```diff
@@ -0,0 +1,27 @@
+import { defineConfig } from "vitest/config";
+import typescript from "@rollup/plugin-typescript";
+import { nodeResolve } from "@rollup/plugin-node-resolve";
+import commonjs from "rollup-plugin-commonjs";
+import peerDepsExternal from "rollup-plugin-peer-deps-external";
+import pluginJson from "@rollup/plugin-json";
+
+/**
+ * @see https://vitejs.dev/config/
+ */
+export default defineConfig({
+  build: {
+    lib: {
+      name: "@latticexyz/react",
+      entry: "src/index.ts",
+      formats: ["cjs", "es"],
+    },
+    outDir: "dist",
+    minify: false,
+    sourcemap: true,
+  },
+  test: {
+    environment: "jsdom",
+    globals: true,
+  },
```

Comment:
this config is the test config too which is very nice

Reply:
Note how I didn't need to change any of the tests.  This is because jest and vitest are almost the same api